.github/workflows/release.yml

```diff
@@ -210,7 +210,7 @@ jobs:
     name: GitHub release
     runs-on: ubuntu-18.04
     environment: build-and-publish
-    needs: [preflight]
+    needs: preflight
```

Comment:
Are you sure removing the brackets is gonna work?

Reply:
Yes, that’s how it’s done at the other places too.
Generally speaking, when a list can be provided, GitHub workflow accepts both a single item or a list. Since we’re using the single item syntax in most place I just unified it.